### ISSUE TITLE
医学数据集测试结果的问题

### ISSUE THREAD

**Issue by Mr-Kerry** [pangdatangtt/UniNet#29]
### 环境
Python 版本: 3.9.5
PyTorch 版本：1.10.1+cu111
显卡: NVIDIA GeForce RTX 4090
操作系统：Ununtu
### 问题
在APTOS、ISIC2018、OCT2017医学数据集上复现的结果与论文中提供的相差过大，使用的参数是代码中默认的，请问是训练医学数据集时需要调整参数吗？
复现结果如下：
| object   |   image_auroc |   acc |    f1 |
|:---------|--------------:|------:|------:|
| APTOS    |         90    | 85.2  | 89.8  |
| ISIC2018 |         82.5  | 78.8  | 73.5  |
| OCT2017  |          95.2 |  91.1 | 94.1  |
### 复现日志
```
APTOS
python main.py --domain medical --dataset APTOS
training on APTOS dataset
domain:medical, setting:oc, dataset:APTOS, epochs:100, batch_size:8, image_size:256, center_crop:256, lr_s:0.005, lr_t:1e-06, T:2, weighted_decision_mechanism:True, default:0.3, alpha:0.01, beta:3e-05, train_and_test_all:False, is_saved:True, save_dir:./saved_results, load_ckpts:False, 
cuda
iters: 250/1000, loss:18.6341
Auroc: 82.28, f1: 85.87, acc: 77.99
modules have been saved to ./ckpts/APTOS/BEST_I_ROC.pth
iters: 500/1000, loss:18.4651
Auroc: 89.10, f1: 89.13, acc: 84.45
modules have been saved to ./ckpts/APTOS/BEST_I_ROC.pth
iters: 750/1000, loss:18.4138
Auroc: 89.57, f1: 89.65, acc: 84.86
modules have been saved to ./ckpts/APTOS/BEST_I_ROC.pth
iters: 1000/1000, loss:18.3851
Auroc: 89.99, f1: 89.75, acc: 85.16
modules have been saved to ./ckpts/APTOS/BEST_I_ROC.pth
training over!
testing on APTOS dataset
cuda
Loading weights from ./ckpts/APTOS/BEST_I_ROC.pth
using UniNet model
| object   |   image_auroc |   acc |    f1 |
|:---------|--------------:|------:|------:|
| APTOS    |         90    | 85.2  | 89.8  |
| mean     |         89.99 | 85.16 | 89.75 |

ISIC2018
python main.py --domain medical --dataset ISIC2018
training on ISIC2018 dataset
domain:medical, setting:oc, dataset:ISIC2018, epochs:100, batch_size:8, image_size:256, center_crop:256, lr_s:0.005, lr_t:1e-06, T:2, weighted_decision_mechanism:True, default:0.3, alpha:0.01, beta:3e-05, train_and_test_all:False, is_saved:True, save_dir:./saved_results, load_ckpts:False, 
cuda
iters: 250/2000, loss:18.6460
Auroc: 75.92, f1: 65.19, acc: 67.36
modules have been saved to ./ckpts/ISIC2018/BEST_I_ROC.pth
iters: 500/2000, loss:18.5216
Auroc: 78.18, f1: 68.24, acc: 72.02
modules have been saved to ./ckpts/ISIC2018/BEST_I_ROC.pth
iters: 750/2000, loss:18.4431
Auroc: 76.68, f1: 68.97, acc: 72.02
iters: 1000/2000, loss:18.2070
Auroc: 76.60, f1: 70.65, acc: 72.02
iters: 1250/2000, loss:18.1825
Auroc: 78.10, f1: 70.86, acc: 73.58
iters: 1500/2000, loss:18.1646
Auroc: 76.42, f1: 70.45, acc: 73.06
iters: 1750/2000, loss:18.1343
Auroc: 82.49, f1: 73.55, acc: 78.76
modules have been saved to ./ckpts/ISIC2018/BEST_I_ROC.pth
iters: 2000/2000, loss:18.1046
Auroc: 81.35, f1: 71.91, acc: 74.09

training over!
testing on ISIC2018 dataset
cuda
Loading weights from ./ckpts/ISIC2018/BEST_I_ROC.pth
using UniNet model

| object   |   image_auroc |   acc |    f1 |
|:---------|--------------:|------:|------:|
| ISIC2018 |         82.5  | 78.8  | 73.5  |
| mean     |         82.49 | 78.76 | 73.55 |

OCT2017
python main.py --domain medical --dataset OCT2017
training on OCT2017 dataset
domain:medical, setting:oc, dataset:OCT2017, epochs:100, batch_size:8, image_size:256, center_crop:256, lr_s:0.005, lr_t:1e-06, T:2, weighted_decision_mechanism:True, default:0.3, alpha:0.01, beta:3e-05, train_and_test_all:False, is_saved:True, save_dir:./saved_results, load_ckpts:False, 
cuda
iters: 250/1000, loss:18.8862
Auroc: 95.20, f1: 94.07, acc: 91.10
modules have been saved to ./ckpts/OCT2017/BEST_I_ROC.pth
iters: 500/1000, loss:18.7676
Auroc: 94.82, f1: 94.77, acc: 92.00
iters: 750/1000, loss:18.7100
Auroc: 91.64, f1: 94.57, acc: 91.60
iters: 1000/1000, loss:18.6709
Auroc: 94.87, f1: 95.40, acc: 93.00

training over!
testing on OCT2017 dataset
cuda
Loading weights from ./ckpts/OCT2017/BEST_I_ROC.pth
using UniNet model

| object   |   image_auroc |   acc |    f1 |
|:---------|--------------:|------:|------:|
| OCT2017  |          95.2 |  91.1 | 94.1  |
| mean     |          95.2 |  91.1 | 94.07 |
